requirements-dev.txt

```diff
@@ -2,3 +2,5 @@
 
 pytest == 5.4.1
 flake8 == 3.7.9
+git+https://github.com/canonical/operator
```

Comment:
should change this to just `ops` when we release 0.7 (or drop the `harness.cleanup` code in the test until 0.7 is a thing)

Reply:
You don't *need* cleanup unless you're doing resources, but it does feel better to have it in case we add something else in the future.

Reply:
dropped cleanup for now